docs/devices.md

```diff
@@ -1,10 +0,0 @@
-Devices
-===
-
-See:
-
-* http://api.zeromq.org/2-1:zmq-device
-* `IDevice` and `DeviceBase`
-* `ForwarderDevice`
-* `QueueDevice`
-* `StreamerDevice`
```

Comment:
Why delete this file?

Reply:
I can recover it, it wasn't linked in the sidebar and with the current content, I thought it was an incomplete page to be honest.